site/_scss/blocks/_language-list.scss

```diff
@@ -0,0 +1,4 @@
+.language-list a {
+  color: var(--color-secondary-text);
+  margin-inline-start: 3px;
```

Comment:
This is here because you seem to not have whitespace between the language choices, so the actual HTML looks like:

```html
Language (foo),Language (bar)
```

... maybe fix the HTML generation instead?